### DISCUSSION TITLE
Are synonyms going to be included in 4.x?

### DISCUSSION THREAD

**Discussion by bryllyg** [microsoft-search/pnp-modern-search#2356] [Ideas]
**Is your feature request related to a problem? Please describe.**
Hi. A quick query on the status of a feature...The synonyms feature in 3.x is very useful. I don't see the feature in the current preview of 4.x. Is there a plan to include this later? or an alternate path for this functionality?

**Describe the solution you'd like**
Continuation of 3.x feature. It would be lovely to have the ability to import this data, but even having to input by hand is good.

**Comment by FranckyC** [comment]
No planned yet sorry. I'll add it as an enhancement.

**Comment by bryllyg** [comment]
Sorry! Didn't mean to close this. Wrong button!

I was about to ask if there was any update on if this feature might make it into future releases. We would really like to move along to 4.x so we can use the fabulous new connectors, but are hampered by the need for some way to provide synonym support. Thanks for any updates.

**Comment by wobba** [comment]
If someone have time to implement it, we can add it. Both @FranckyC and I have been busy with regular work lately, not having time to spend here :)

**Comment by bryllyg** [comment]
Perfectly understandable! You all seem like you keep a huge number of projects in the air at one time! :-) I'm also asking the Topic Center and Microsoft Search UX Customization teams about their plan forward for synonyms. Thanks!

**Comment by sliziky** [comment]
Hi, any updates on the synonyms? Is there any workaround?

**Comment by diplobelsdb** [comment]
Implementing Synonyms in V4 would be a great improvement. Thanks in advance!

**Comment by mitehe1983** [comment]
Hello,
do you have any updates about the synonyms in V4? any workaround?
Thank you in advance

**Comment by RamBalani** [comment]
We are using PnP Modern Search web-part V4 for searching US FDA content on our SmartSearch app.
We have now clubbed an OpenAI  ChatGPT-3.5-Turbo Powerapps with SmartSearch (full text PnP Web search web part)  and the combo works great.

However I concur with the need and discussions here -- that adding synonym capability to the PnP Modern Search web parts would be vast improvement!
Looking forward to that - any ETAs on that ?
As it is today-  OpenAI GPTChat implementation outshines us with search results by simply associating the term 'biosimilar' with 'biologics' -thought I'd share this.
